sheer/query.py

```diff
@@ -304,8 +306,23 @@ def get_document(doctype, docid):
         raw_results = es.get(index=es_index, doc_type=doctype, id=docid)
         return QueryHit(raw_results)
 
+    def convert_to_datetime(timestamp):
```

Comment:
This is what we came up with:
```
    def convert_to_datetime(timestamp):
        date = parser.parse(timestamp)
        if not date.tzinfo:
            date = date.replace(tzinfo=timezone('America/New_York'))
        return date.astimezone(timezone('UTC'))
```

That way, we set the timezone to be EST if it's not set, then convert it to UTC, to be able to compare it to datetime.now() which will also be converted to UTC.